src/Files.App/Helpers/AppLifecycle.cs

```diff
@@ -0,0 +1,360 @@
+﻿using Files.App.Services.Settings;
+using Microsoft.Windows.AppLifecycle;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.MemoryMappedFiles;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.Json;
+using Files.App.UserControls.MultitaskingControl;
+using Files.App.Helpers;
+
+namespace Files.App.Helpers
+{
+	public class AppLifecycle
+	{
+		public static string SharedMemoryNameHeader = "FilesAppTabsWithID";
+		public static MemoryMappedFile? SharedMemoryNameHeaderMemory;
+		public static string SharedMemoryName = SharedMemoryNameHeader + defaultBufferSize.ToString();
```

Comment:
`private static string defaultSharedMemoryName = sharedMemoryHeaderName + defaultBufferSize.ToString();`
Use `defaultSharedMemoryName` instead of `sharedMemoryHeaderName + defaultBufferSize.ToString();`